src/bundle/Resources/views/themes/admin/ui/menu/user.html.twig

```diff
@@ -5,25 +5,43 @@
     {% set user_image = user.content.thumbnail %}
 
     <div class="ibexa-header-user-menu">
-        <img 
-            class="ibexa-header-user-menu__image" 
-            src="{{ user_image.resource }}" 
-            alt="{{ user.name }}" 
+        <div
+            class="ibexa-header-user-menu__notifications-toggler"
             data-bs-toggle="modal"
             data-bs-target="#view-notifications"
-        />
-        <span
-            class="ibexa-header-user-menu__notice-dot ibexa-header-user-menu__notice-dot--no-notice"
-            data-bs-toggle="modal"
-            data-bs-target="#view-notifications"
-        ></span>
-        <button class="ibexa-header-user-menu__name">{{ user.name }}</button>
+        >
+            <svg class="ibexa-icon ibexa-icon--small ibexa-icon--light">
```

Comment:
Question: are we still using `ibexa-icon--light`? I thought we don't as I see it very rarely in recent PRs.